src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -194,12 +231,12 @@ object HeaderPlugin extends AutoPlugin {
             groupedFiles.flatMap(checkHeader(fileType, commentStyle))
         }
     if (filesWithoutHeader.nonEmpty)
-      log.error(
+      throw new MessageOnlyException(
```

Comment:
How does that change the user experience?

Reply:
This fails the task, leading to the text being logged at error, previously the check task would complete successfully _and_ log errors. Not sure how that could have ever worked with the scripted tests though, because failing the task is how sbt knows something went wrong.

Reply:
Thanks.